README.md

```diff
@@ -52,37 +52,39 @@ In addition, certain aspects can be overridden dynamically at runtime from a JSO
+| `DYNAMIC_CONFIG_FILE` | Location of an optional external config file that provides configuration at runtime.                                                                                                                                                                                                                                                                                                                                                                                  | empty                   |                                  |
+| `DYNAMIC_CONFIG_WATCHER_INTERVAL` | Interval that dynamic config file is examined for changes in content (in ms)                                                                                                                                                                                                                                                                                                                                                                                          | `30000`                 |                                  |
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | Activate OTLP tracing                                                                                                                                                                                                                                                                                                                                                                                                                            | empty                   |                                  |
+| `OTEL_EXPORTER_JAEGER_ENDPOINT` | Activate Jaeger tracing                                                                                                                                                                                                                                                                                                                                                                                                                           | empty                   |                                  |
+| `ACTIVATE_TRACING` | Activate tracing. When true  OTEL_EXPORTER_OTLP_ENDPOINT or OTEL_EXPORTER_JAEGER_ENDPOINT                                                                                                                                                                                                                                                                                                                                                                                                                            | `false`                   | `activateTracing`                |
```

Comment:
why all this big "green" change when you have just added a few env vars at the bottom?

Reply:
i updated the readme